.github/workflows/zizmor.yaml

```diff
@@ -0,0 +1,32 @@
+name: zizmor workflow audits
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  zizmor-workflow-audits:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: zizmor workflow audits
```

Comment:
This step should be moved to `ci_file_health.yaml` instead of an independent file.